HelgaShiryaeva/assignment1/ReversedListAccessUtil

```diff
@@ -0,0 +1,91 @@
+public class ReversedListAccessUtil {
+    public <T> T getReversed(int index, MyList<T> list) {
+      return list.get(list.size() - index);
```

Comment:
Clever use of size :). How would you implement it if you could not store size of the list?